test/benchmark/iperf3/tcp_virtio_bw/host.sh

```diff
@@ -14,6 +14,6 @@ stop_guest() {
 trap stop_guest EXIT
 
 # Run iperf3 client
-/usr/local/benchmark/iperf/bin/iperf3 -c 127.0.0.1 -f m
+/usr/local/benchmark/iperf/bin/iperf3 -c 10.0.2.15 -f m
```

Comment:
I guess that now is time to define an env variable for the IP address. This address is repeated all over the benchmark tests.